README.md

```diff
@@ -701,6 +701,15 @@ type.name = "pug"
 
 ```
 
+Once inline tables are defined, their values are fully set, and new elements and
```

Comment:
"Element" only occurs in reference to Arrays in TOML, but I don't think you intended to reference Arrays, so please use "keys" instead of "elements"?

Reply:
I get so sick of saying "Key/Value Pair" inside of tables. That must be why I said "element" instead. Since this is a clarification update and doesn't slog down the wording (in fact, it tightens it up a bit), I'll gladly make the change that you suggest.

I could support a lawyerly rewrite of the spec so that all terms are rigidly defined. I don't think it was originally written that way, though, and I'm a little afraid that the strident tone of the original might get lost in such an effort. But TOML is catching eyes because it's well-defined. So this is a step in the right direction.